### DISCUSSION TITLE
How to prevent push notification from being dismissed after being tapped?

### DISCUSSION THREAD

**Discussion by esdeguzman** [expo/expo#17190] [Archived: Questions]
I was reading the Expo Notifications documentation and came across **Notifications.useLastNotificationResponse();**, I was successfully catching the tapped event but have no way to prevent the (tapped) push notif do be dismissed.

Is there anything I missed in the documentation?

Thank you!

**Comment by alexanderdavide** [comment]
I'm facing the same issue right now. Have you found a solution meanwhile?

**Comment by raajnadar** [comment]
Did you try using `sticky: true`?

https://docs.expo.dev/versions/latest/sdk/notifications/#notificationcontentinput

**Comment by SethCram** [comment]
The docs confused me quiet a bit, but after some testing setting `autoDismiss: false` worked for me on expo 48.0.21
